### PR TITLE
added to the ReadMe file-close

### PR DESCRIPTION
**Pull request by sruble**

### CODE REVIEW

**Review by weberwilliamb — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,2 +1,42 @@
+![logo](photo_points/wwwroot/images/fncf-rgb-horizontal-logo_1.jpg)
+
 # photo-points
-Web Application for Photo Points data and API
+Web Application for Photo Points data and API. 
+The web application, created for _Friends of North Creek Forest_, will allow the administrator to view approved photos, search photos, view pending photos from the mobile application.
```
Comment (Bansenauer-Cascadia) [inline README.md]: The last part "view pending photos from the mobile application" is unclear. I think you may mean,  "... and view pending photos, submitted from the mobile app, for approval to post to the FNCF Photo Points website gallery.

**Review by Bansenauer-Cascadia — CHANGES_REQUESTED** [review]
Great draft of your README, just a small wording change suggested here.

File: README.md (hunk 1 above)
Comment (sruble) [reply README.md]: yes, that sounds a lot clearer, I've updated the change

**Comment by sruble** [comment]
Once this branch is merged to develop, we can proceed with a merge to the master branch.

**Pull request merged**